Dockerfile

```diff
@@ -6,6 +6,6 @@ RUN mv /usr/bin/gather /usr/bin/gather_original
 # Use our gather script in place of the original one
 COPY gather_cnv /usr/bin/gather
 COPY gather_cnv_* /usr/bin/
-COPY node-gather/node-gather-* /etc/
+COPY node-gather/node-gather-*.yaml /etc/
```

Comment:
If `make docker-push` is guaranteed to never be done manually, then this is fine. But if there's a chance of people doing manual pushes on occasion, then I'd replace the glob with an explicit list of yamls, to prevent someone pushing a borked build due to not doing `make manifests` first.

(Or add the file dependencies into the Makefile, but I think it's easier here.)

Reply:
I wonder if we need to include the manifests in the container at all...

Reply:
Replaced with list of yamls.

The manifest is needed to create a daemonset which collects data from node.
The first image is the must-gather image. We run a single pod for this. This collects the "gather data" using mostly the oc tool. Note that it has no access to nodes. In order to collect data from nodes, this pod (the must-gather pod) creates a daemonset that can access data on nodes (hostNetwork). 
The manifest here contains the ds defintion.